api/openapi.yaml

```diff
@@ -500,3 +519,29 @@ components:
           example: "this is an error message"
       required:
         - message
+
+    Timestamp:
+      type: string
+      description: Date and time (RFC 3339) representing the UTC timestamp of the health check
+      example: 2021-10-23T15:05:37Z
+
+    GoodHealthCheckResponse:
```

Comment:
Something we should cleanup.

Currently our task endpoints also include the error block, we just expect this to be empty when a response is successful. I think the original intent behind this was to allow for a single object to be consumed by a client. We should decide if this is the way we'd like to handle response objects going forward, or if we want two distinct objects (one for success and one for errors).

Reply:
TODO: create spike ticket to re-evaluate this

Reply:
If we add error here, there's not a lot of value in having two return types `Good...` and `Bad...`